tests/test_config.py

```diff
@@ -157,7 +157,8 @@ def test_invalid_config_options_output():
 
 
 @pytest.mark.parametrize(
-    "key, value", list(DEFAULTS.items()),
+    "key, value",
```

Comment:
Hmm... I don't know what's going on here. I thought Black would've formatted it to looks similar? I mean, there are no changes here.

Reply:
Don't worry about the black problems. I'll pin the black version to an earlier version and you can rebase from master if you don't mind. No hurry though, I won't have time to look at your changes before the weekend.